argonaut/src/main/scala/org/http4s/argonaut/ArgonautInstances.scala

```diff
@@ -16,13 +16,19 @@ trait ArgonautInstances {
       )
     }
 
-  implicit val jsonEncoder: EntityEncoder[Json] =
+  def jsonEncoder(prettyParams: PrettyParams): EntityEncoder[Json] =
     EntityEncoder.stringEncoder(Charset.`UTF-8`).contramap[Json] { json =>
       // TODO naive implementation materializes to a String.
       // Look into replacing after https://github.com/non/jawn/issues/6#issuecomment-65018736
-      Argonaut.nospace.pretty(json)
+      prettyParams.pretty(json)
     }.withContentType(`Content-Type`(MediaType.`application/json`, Charset.`UTF-8`))
 
+  implicit val jsonEncoder: EntityEncoder[Json] =
+    jsonEncoder(Argonaut.nospace)
```

Comment:
I would want to be able to configure this one as well.

It may end up making sense to have the encoders defined in a class that takes a `PrettyParams` as an input, I don't know.